src/components/TodoAppTodo.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo,
+  deleteHandler?: (todoId: number) => void,
+  isProcessed: boolean,
+  onUpdate?: (todoId: number, title?: string) => void,
+};
+
+export const TodoAppTodo: React.FC<Props> = ({
```

Comment:
`TodoItem` name will be more understandable

Reply:
And you may remove `TodoApp` prefix from `Header` and `Footer` - that's all `TodoApp`